### DISCUSSION TITLE
The dist does not match the latest code it seems

### DISCUSSION THREAD

**Discussion by dgorti** [vakata/jstree#2486] [General]
Ivan,
I created a PR to make some changes to JSTree to make it a bit more compliant for WCAG. One thing  noticed is that what is in dist does not seem to be the latest code from (master). Could you please provide some guidance here?

**Comment by vakata** [comment]
My guess is I merged a PR and forgot to build after that (most PRs do not include built files). I will fix this today - thank you for letting me know.

**Comment by vakata** [comment]
Updated.
